.github/ISSUE_TEMPLATE/bug_report_template.md

```diff
@@ -12,9 +12,14 @@ assignees: ''
 Thank you for submitting an issue. Please refer to our [issue policy](https://www.github.com/voxel51/fiftyone/blob/develop/ISSUE_POLICY.md) for information on what types of issues we address.
 
 1. Please fill in this template to ensure a timely and thorough response
-2. Place an "x" between the brackets next to an option if it applies. For example:
+2. Remove the section instructions but leave the section header
+3. Place an "x" between the brackets next to an option if it applies. For example:
     -   [x] Selected option
-3. **Please delete everything above this line before submitting the issue**
+4. **Please delete everything above this line before submitting the issue**
+
+### Describe the problem
```

Comment:
Move description to the top so it's the first thing you see on the page or github email.

Reply:
Now, you have to scroll a ways down sometimes (especially on mobile) to get to the crux of the issue, you've got the system info upfront which is relevant but relatively unimportant.